test/controllers/agentAppointment/AppointAgentControllerSpec.scala

```diff
@@ -439,7 +439,7 @@ class AppointAgentControllerSpec extends VoaPropertyLinkingSpec with MockitoSuga
 
     val page = HtmlPage(Jsoup.parse(contentAsString(res)))
     page.mustContainText("Select one or more properties")
-    page.titleMustMatch("Error: Which properties should the agent be removed from - Valuation Office Agency - GOV.UK")
+    page.titleMustMatch("Error: Remove agent gg-ext-id from one or more properties - Valuation Office Agency - GOV.UK")
```

Comment:
I guess `gg-ext-id` is defined somewhere (in FakeObjects, as `ggExternalId`). In which case, this would be more stable:
```
page.titleMustMatch(s"Error: Remove agent $ggExternalId from one or more properties - Valuation Office Agency - GOV.UK")
```